packages/cyberstorm/src/components/TextInput/TextInput.tsx

```diff
@@ -4,14 +4,19 @@ import styles from "./TextInput.module.css";
 import { Icon } from "../Icon/Icon";
 import { classnames } from "../../utils/utils";
 
-export interface TextInputProps {
+export interface TextInputProps
+  extends React.ComponentPropsWithRef<"input">,
```

Comment:
FYI opening the create form dialog on teams-page now shows this in console:

```
Warning: TextInput: `ref` is not a prop. Trying to access it will result in `undefined` being returned.
If you need to access the same value within the child component, you should pass it as a different
prop. (https://reactjs.org/link/special-props)
...


Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you
mean to use React.forwardRef()?
Check the render method of `CreateTeamForm`.
…
```

Don't know if it's related to this change, but it seems likely?

Reply:
Fixd